ci-operator/config/openshift/release/openshift-release-master.yaml

```diff
@@ -17,6 +17,11 @@ tests:
   commands: make check-services
   container:
     from: src
+- as: ipi-deprovision-aws
+  cron: 45 4,10,16,22 * * *
```

Comment:
I dunno what the format here is.  Any suggestions for what the content should be?

Reply:
I've pushed 4234ae43de -> d4fb511e9b to run this at X:51, every even hour.  Not sure if there's a convenient way to avoid overlapping multiple runs, but with 10 clusters getting reaped in parallel, 30m per-cluster caps, and 2h between pruner jobs, we can have a fair amount of leakage before we risk overlapping pruners.